batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java

```diff
@@ -251,6 +258,68 @@ public void createBatchCustomEventTest()
                     .anyMatch(runnable -> runnable.getDisplayName().equals(displayName))));
   }
 
+  @Test
```

Comment:
same comment as in the other PR. Kindly look at the changes unrelated to this PR and remove them.

Reply:
In this PR I added only one test for Batch allocation labels. The last commit was  "Merged changes from main, resolved merge conflict", to keep my branch updated. Please review my code once more.